pkg/shp/streamer/streamer.go

```diff
@@ -1,10 +1,12 @@
 package streamer
 
 import (
+	"fmt"
 	"io"
 	"os"
 	"sync"
 
+	progressbar "github.com/schollz/progressbar/v3"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
```

Comment:
Nit: I think we never wrote it down, but there was the idea to sort k8s imports separately.
```suggestion
	"k8s.io/cli-runtime/pkg/genericclioptions"
	"k8s.io/client-go/kubernetes"
	"k8s.io/client-go/rest"

	progressbar "github.com/schollz/progressbar/v3"
```